src/h3/mod.rs

```diff
@@ -406,6 +406,8 @@ pub struct Config {
     max_header_list_size: Option<u64>,
     qpack_max_table_capacity: Option<u64>,
     qpack_blocked_streams: Option<u64>,
+    dgram_poll_threshold: u64,
+    stream_poll_threshold: u64,
```

Comment:
Does it need to be `u64`?

Reply:
That's what @ghedo suggested in https://github.com/cloudflare/quiche/pull/704#discussion_r519893116